src/codegen/expression.rs

```diff
@@ -2186,6 +2186,56 @@ fn expr_builtin(
                 expr: Box::new(codegen_expr),
             }
         }
+        ast::Builtin::ECRecover => {
```

Comment:
Is it possible to use `Instr::Unimplemented` here?

Reply:
Why `Instr::Unimplemented` preferable over an actual implementation?

Reply:
Which test coverage numbers?

Reply:
Yes, yolo in code in which we can't test isn't a good practice

Reply:
ok, I've been outvoted, it's gone

Reply:
I mean it could also be commented out, saying that this is how we think it should be implemented, but it's commented out and an `Unreachable` instead until we can test it?

Reply:
Good idea. I've added the code I wrote as a comment